enterprise/frontend/src/metabase-enterprise/license/components/LicenseAndBillingSettings/LicenseAndBillingSettings.tsx

```diff
@@ -45,13 +44,16 @@ const getDescription = (tokenStatus?: TokenStatus, hasToken?: boolean) => {
     );
   }
 
-  const daysRemaining = moment(tokenStatus.validUntil).diff(moment(), "days");
+  const daysRemaining = moment(tokenStatus["valid-thru"]).diff(
+    moment(),
+    "days",
+  );
```

Comment:
I'm working on this same thing in another part of the app, and am using `dayjs`. So I might extract a helper/util to handle that part.

Also, as Ryan noted in another PR of mine, `tokenStatus` is set in UTC, and moment/dayjs will grab the current timestamp in user's browser locale. But I don't think you need to worry about that in this PR.